### ISSUE TITLE
GKs not considered in trackable amount in lists

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#4802]
Possibly a side effect of #4779 

2015.03.30-NB:
- Store GC3FJ5F for offline use
- In the stored list it is shown that 2 trackables are in that cache
- Open the cache inventory

--> The cache contains 2 TB and 1 GK, so I would expect 3 in the summary on the list

**Comment by kumy** [comment]
Store GK in Datastore would help for that, for stored cache only.

But in a list view like "nearby", if we want to display GK count, we have to do requests for each cache ID sequentially.

Note: with GKMap I can implement something to request for exemple, GK by zone and do a matching process with currently listed caches...

**Comment by Lineflyer** [comment]
> Note: with GKMap I can implement something to request for exemple, GK by zone and do a matching process with currently listed caches...

Isn't is possible to do one request with a collection of waypoints in it and return one xml with all content.
GCVote does it like that.

**Comment by kumy** [comment]
The GK Api doesn't provide that (from memory), maybe we should ask Filip.
For GKM I can implement that.

**Comment by kumy** [comment]
Collection request is implemented in GKM Api
Ex: http://api.geokretymap.org/export2.php?wpts=GC5GQXH,GC5QPBK,,GC5BYWV,GC2DX0V,FOOBAR

Request is limited to the first 20 tokens at the same time.

I have some test code in branches:
https://github.com/kumy/cgeo/commits/issue-4802
https://github.com/kumy/cgeo/commits/issue-4802-bis

**Comment by Lineflyer** [comment]
@kumy: Did you suggested code for this issue come into `master`  at some point of time. I can't remember.

**Comment by kumy** [comment]
I don't think it has ever been merged in master. And I can't remember why I didn't go further.. I have to check what's different in the two branches...
